### DISCUSSION TITLE
shouldn't in_network files listed in the TOC be directly downloadable?

### DISCUSSION THREAD

**Discussion by nmaylevc** [CMSgov/price-transparency-guide#527] [Q&A] [answered]
In a table of contents file there is a list of in_network_files with locations that are URLs.
My expectation was that these URLs could be downloaded. For some files when one downloads one of these URLs one doesn't get a data file, but instead one gets a webpage that has the file listed on it.  

Shouldn't the in_network_file just be easily and directly downloadable?

The same issue occurs with the allowed_amount_file.

What is the requirement?

**Comment by shaselton-usds** [comment, marked as answer]
@nmaylevc There is quite a bit of information for the Table of Content files covered in a [previous webinar](https://www.cms.gov/healthplan-price-transparency/resources/webinars). Take a look at November 30th for table of contents information.

**Reply by nmaylevc** [reply]
I just re-watched the webinar. In the last webinar, you clearly stated and wrote on a slide that "External In-Network Files: Must be valid (comply with the in-network schema) in-network files"

Without naming an insurer, there are TOC files put up by at least one insurer that has URLs that link a page that list a set of in-network-files rather that to a file. From the webinar this is clear invalid.

Perhaps some folks haven't watched the webinars.

How should we report these?

**Reply by JR-CMS** [reply]
You may file a complaint through the webform available on our website, here: https://www.cms.gov/healthplan-price-transparency/contact-us
